pylot/perception/detection/lanenet_detection_operator.py

```diff
@@ -15,6 +15,7 @@
 
 import tensorflow as tf
 
+tf.compat.v1.disable_eager_execution()
```

Comment:
Could you move this into the `__init__` method, just above `pylot.utils.set_tf_loglevel(logging.ERROR)`?